app/src/Config/Config.ts

```diff
@@ -63,6 +63,7 @@ export default class Config {
           codeforces: "54",
           atcoder: "4003",
           omegaup: "cpp17-gcc",
+          szkopul: "-",
```

Comment:
please add the language aliases, even if they are automatically inferred by the judge